### ISSUE TITLE
Menu Accessibility: Focus events not triggering and names are not announced.

### ISSUE THREAD

**Issue by JakeRad** [electron/electron#10507]
* Electron version: 1.7.2
* Operating system: Windows 10 Pro

### Expected behavior

Menu item's are announces by narrator. 

### Actual behavior

Up/Down Arrow is announced instead of names.

### How to reproduce

1. Create Electron App With Menu
2. Open Electron
3. Attempt to navigate Menu with Narrator
4. Observe behavior

**Comment by bartlomiejcieszkowski** [comment]
I checked this issue on:
 Windows 10 Pro 10.0.16299 Build 16299
and it reproduces - it seems that the actual obj_id that we get in L_PARAM is 0xFFFFFFE7,

also the IsScreenReaderActive() check fails - i cross referenced this with chromium source code and found the following comment in the codebase:

// (1) On windows many screen reader detection mechinisms will give false
// positives like relying on the SPI_GETSCREENREADER system parameter. In Chrome
// we attempt to dynamically detect a MSAA client screen reader by calling
// NotifiyWinEvent in NativeWidgetWin with a custom ID and wait to see if the ID
// is requested by a subsequent call to WM_GETOBJECT.
from:
https://github.com/chromium/chromium/blob/f18e79d901f56154f80eea1e2218544285e62623/content/browser/accessibility/browser_accessibility_state_impl.h

**Comment by sofianguy** [comment]
Thank you for your issue!

We haven't gotten a response to our questions in our comment above. With only the information that is currently in the issue, we don't have enough information to take action. I'm going to close this but don't hesitate to reach out if you have or find the answers we need, we'll be happy to reopen the issue.
